components/media-feature/Component.jsx

```diff
@@ -39,49 +40,49 @@ export default function MediaFeature({
   return (
     <Container width="full" paddingX={false}>
       <section className="su-py-45 su-px-20 su-flex su-justify-center su-relative md:su-py-72 md:su-px-50">
-        <div className="su-max-w-[1086px] su-flex su-flex-col su-items-center su-z-[2] su-relative su-p-38 before:su-content-[''] before:su-bg-foggy-light before:su-w-full before:su-h-full before:su-opacity-90 before:su-absolute before:su-z-[-1] before:su-top-0 before:su-left-0 md:su-flex-row md:su-gap-20 md:su-items-start lg:su-p-48 lg:su-gap-48">
```

Comment:
For our `su-stretched-link` to work we need `su-relative` on this outer container of the "card".

Reply:
Oh wait! `su-relative` is already there, nevermind.

However, for group hocus styles to work, add `su-group` class here.

Reply:
I wasn't quite sure where you were saying that `su-group` should go, but I didn't change where it already was, and the hocus styles seem to work.